### PR TITLE
feat: Fix Avatar sizes, add inverse and backgroundColor props

### PR DESCRIPTION
**Pull request by sbrandwoo**
## 🖼 Context

The original sizes of the Avatar didn't match Figma, so these have been updated.

I also added two properties:
- `inverse`: For when the avatar is on a dark background, this changes the inset border colour, and the default background colour
- `backgroundColor`: Optional colour to use instead of the defaults

## ✅ Checklist

- [x] Changes are tested and all tests pass
- [x] Changes meet [accessibility standards](https://github.com/transferwise/neptune-web/blob/master/ACCESSIBILITY.md) and there are no violations in the console
- [x] Changes work in all supported browsers (don't forget IE11)
- [x] You've updated the documentation if necessary

### CODE REVIEW

File: packages/components/src/avatar/Avatar.js [hunk 1]
```diff
@@ -35,13 +38,17 @@ Avatar.propTypes = {
   ]),
   children: Types.node,
   outlined: Types.bool,
+  inverse: Types.bool,
```
Comment (rerdos) [inline packages/components/src/avatar/Avatar.js:41]: I wonder if this should be `theme` to be aligned with other components.

Like: `theme={FlowNavigation.Theme.LIGHT}`

Comment (sbrandwoo) [reply packages/components/src/avatar/Avatar.js]: Done!

File: packages/components/src/avatar/Avatar.js [hunk 2]
```diff
@@ -2,21 +2,25 @@ import React from 'react';
...
 import classnames from 'classnames';
 
-import './Avatar.css';
+import { Size, Theme } from '../common';
 
-import { Size } from '../common';
+import './Avatar.css';
 
-const Avatar = ({ size, type, children, outlined, className }) => (
+const Avatar = ({ size, type, children, outlined, theme, backgroundColor, className }) => (
```
Comment (dozortsev) [inline packages/components/src/avatar/Avatar.js:9]: can we avoid creating `backgroundColor` prop and use `className` one for this purposes?

```jsx
<Avatar .. className="custom-background-color" />
```

Comment (dozortsev) [reply packages/components/src/avatar/Avatar.js:9]: note: this CSS class then can be reused for other `Avatar`s

Comment (sbrandwoo) [reply packages/components/src/avatar/Avatar.js:9]: We're setting colours programmatically, so CSS classes are no good to us.

If we don't want the component to support the use case then we'd need to put the background in the content instead. I'd prefer if the component did what we needed.

Comment (sbrandwoo) [reply packages/components/src/avatar/Avatar.js:9]: Setting a background colour is a product feature that Avatar needs to support.

Comment (dozortsev) [reply packages/components/src/avatar/Avatar.js:9]: okey

**Pull request merged**